tests/test_data/collection_exercise/collection_exercise_details_no_ci.json

```diff
@@ -0,0 +1,36 @@
+{
+  "collection_exercise": {
+    "actualExecutionDateTime": "2017-11-10T12:14:41.768+0000",
+    "actualPublishDateTime": "2017-11-10T12:16:28.509+0000",
+    "caseTypes": [
+      {
+        "actionPlanId": "e71002ac-3575-47eb-b87f-cd9db92bf9a7",
+        "sampleUnitType": "B"
+      },
+      {
+        "actionPlanId": "0009e978-0932-463b-a2a1-b45cb3ffcb2a",
+        "sampleUnitType": "BI"
+      }
+    ],
+    "executedBy": null,
+    "exerciseRef": "221_201712",
+    "id": "14fb3e68-4dca-46db-bf49-04b84e07e77c",
+    "name": "000000",
+    "periodEndDateTime": "2017-09-15T22:59:59.000+0000",
+    "periodStartDateTime": "2017-09-14T23:00:00.000+0000",
+    "scheduledEndDateTime": "2018-06-29T23:00:00.000+0000",
+    "scheduledExecutionDateTime": "2017-09-10T23:00:00.000+0000",
+    "scheduledReturnDateTime": "2017-10-06T00:00:00.000+0000",
+    "scheduledStartDateTime": "2017-09-11T23:00:00.000+0000",
+    "state": "PUBLISHED",
+    "surveyId": "cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87"
+  },
+  "survey": {
+    "id": "cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87",
+    "longName": "Business Register and Employment Survey",
+    "shortName": "BRES",
+    "surveyRef": "221"
+  },
+  "collection_instruments": [
+  ]
+}
```

Comment:
No new line at end of file.

Reply:
Don't think it's necessary in this case as it's json and it's only pep8 thing